.github/workflows/build.yml

```diff
@@ -145,6 +190,47 @@ jobs:
           docker push discourse/discourse_test:slim
           docker push discourse/discourse_test:slim-browsers
           docker push discourse/discourse_test:release
+  test_bookworm:
```

Comment:
This will only work on the `main` branch since the image has not been pushed to Dockerhub

Reply:
This is also a simplified version of the [test](https://github.com/discourse/discourse_docker/pull/823/files#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721R152) job.